### DISCUSSION TITLE
Why do I get a `null` error every time I navigate to this specific page?

### DISCUSSION THREAD

**Discussion by gigamesh** [vercel/next.js#65473] [Help]
### Summary

My pages structure is like this:
```
pages/
  evangelize/
    index.tsx
    submit-tweet.tsx
```

And every time I navigate to the /evangelize/submit-tweet page, this `null` error shows up in the console:
https://www.loom.com/share/db5d7a399144407a8dd3ff98fd24c5ce?sid=54b0955e-d78a-4a1b-94c1-c5cadb2979e8

You can see it yourself by going here: [https://farther.social/evangelize/submit-tweet](https://farther.social/evangelize/submit-tweet)

All pages have `getStaticProps`:

![image](https://github.com/vercel/next.js/assets/32403632/81289d49-cb04-49ea-9b46-fb8ddfeaa69d)

It is sending tons of logs to Sentry so I'd really like to figure out what's causing it. Any ideas?

**Comment by icyJoseph** [comment]
There's some logic on your `_app` that triggers that. Namely a `console.error` statement:

```js
        function c() {
            let {error: e, ...t} = arguments.length > 0 && void 0 !== arguments[0] ? arguments[0] : {}
              , {toast: n} = (0,
            u.p)()
              , c = o(t)
              , d = s(e=>{
                var t;
                let {error: a, capture: o=!0, extra: s, toastMsg: u, toastDuration: c=5e3, showGenericToast: d, level: f} = e;
                console.error(a); // The `null` comes from here.
```
